graylog2-web-interface/src/components/datanode/DataNodeConfiguration/CAConfiguration.tsx

```diff
@@ -21,34 +21,51 @@ import { Tabs, Tab, Alert } from 'components/bootstrap';
 import CACreateForm from 'components/datanode/DataNodeConfiguration/CACreateForm';
 import CAUpload from 'components/datanode/DataNodeConfiguration/CAUpload';
 import DocumentationLink from 'components/support/DocumentationLink';
+import useSendTelemetry from 'logic/telemetry/useSendTelemetry';
+import { TELEMETRY_EVENT_TYPE } from 'logic/telemetry/Constants';
 
 const StyledAlert = styled(Alert)`
   margin-top: 10px;
   margin-bottom: 10px;
 `;
 
 const TAB_KEYS = ['create', 'upload'];
-const CAConfiguration = () => (
-  <>
-    <h2>Configure Certificate Authority</h2>
-    <p>
-      In this step you can either upload or create a new certificate authority.<br />
-      The certificate authority will provision and manage certificates for your Data Nodes more easily.
-    </p>
-    <StyledAlert bsStyle="info" title="Reusing certificates">
-      If your existing cluster uses certificates, by default these will get replaced with the Graylog CA
-      and automatically generated certificates during provisioning of the data nodes in the next step.
-      If you want to include your own CA, you can upload an existing certificate.
-      Please see <DocumentationLink page="graylog-data-node" text="Graylog Data Node - Getting Started" /> for more information.
-    </StyledAlert>
-    <Tabs defaultActiveKey={TAB_KEYS[0]} id="ca-configurations">
-      <Tab eventKey={TAB_KEYS[0]} title="Create new CA">
-        <CACreateForm />
-      </Tab>
-      <Tab eventKey={TAB_KEYS[1]} title="Upload CA">
-        <CAUpload />
-      </Tab>
-    </Tabs>
-  </>
-);
+
+const CAConfiguration = () => {
+  const sendTelemetry = useSendTelemetry();
+
+  const handleTabSwitch = (e) => {
+    sendTelemetry((e?.target?.innerText === 'Upload CA')
```

Comment:
Would be nice to move `Upload CA` into a constant and reuse it at line 63 to avoid bugs.